src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Serializer/ProtobufOtlpMetricSerializer.cs

```diff
@@ -0,0 +1,352 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.Serializer;
+
+internal class ProtobufOtlpMetricSerializer
```

Comment:
Can this be made `internal static class`? It looks like that's what we have for `ProtobufOtlpTraceSerializer`.

Reply:
yup, that's a miss on my part. easy fix